src/main/java/nextstep/common/constant/Type.java

```diff
@@ -0,0 +1,25 @@
+package nextstep.common.constant;
+
+public enum Type {
+    DISTANCE("distance"),
+    DURATION("duration");
+
```

Comment:
경로 가중치에 대한 타입이니 이름을 구체화해보면 어때요? 
```suggestion
public enum PathType {
    DISTANCE("distance"),
    DURATION("duration");

```